packages/kit/src/core/adapt/builder.js

```diff
@@ -229,6 +234,37 @@ export function create_builder({
 
 		writeServer(dest) {
 			return copy(`${config.kit.outDir}/output/server`, dest);
+		},
+
+		hasServerTracingFile() {
+			return existsSync(`${config.kit.outDir}/output/server/tracing.server.js`);
+		},
+
+		trace({
+			entrypoint,
+			tracing,
+			start = join(dirname(entrypoint), 'start.js'),
+			tla = true,
```

Comment:
What's the difference between using top-level await and not using it? I can see that it's been set to false for all the edge environments we're deploying to but I think they do support it. Does it have some other affect on the tracing when enabled in those edge environments?

Reply:
Vercel Edge was erroring out when I tried to deploy anything with TLA. Netlify was erroring out locally, though it could've been a skill issue. I'd welcome your help testing!

Reply:
Awesome! Thank you